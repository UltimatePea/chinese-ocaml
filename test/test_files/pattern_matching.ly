「：骆言测试程序：模式匹配：」

夫 测试数字 者 受 x 焉 算法 乃
  观 x 之 性
  若 0 则 答 『零』
  若 1 则 答 『一』
  若 2 则 答 『二』
  余者 则 答 『其他』
  观毕 是谓

打印 (测试数字 0)
打印 (测试数字 1)
打印 (测试数字 2)
打印 (测试数字 5)