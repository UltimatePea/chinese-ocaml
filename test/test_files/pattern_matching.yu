// 骆言测试程序：模式匹配
// Luoyan Test Program: Pattern Matching

让 测试数字 = 函数 x ->
  匹配 x 与
  | 0 -> "零"
  | 1 -> "一"
  | 2 -> "二"
  | _ -> "其他"

打印 (测试数字 0)
打印 (测试数字 1)
打印 (测试数字 2)
打印 (测试数字 5)