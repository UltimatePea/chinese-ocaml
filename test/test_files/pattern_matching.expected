零
一
二
其他